hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -356,6 +357,63 @@ def hpat_pandas_series_append_impl(self, to_append):
     return hpat_pandas_series_append_impl
 
 
+@overload_method(SeriesType, 'min')
+def hpat_pandas_series_min(self, axis=None, skipna=True, level=None, numeric_only=None):
+    """
+    Pandas Series method :meth:`pandas.Series.min` implementation.
+
+    .. only:: developer
+
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_attr3
```

Comment:
Specified incorrect name of the test.

Reply:
Why incorrect? This test alredy existed, and it's chech series.min

Reply:
Specified test `test_series_attr3` checks series.size.